dashboards/Node_Summary.json

```diff
@@ -1001,30 +1001,241 @@
+                    "sanitizeURLEnabled": false,
+                    "scaledDecimals": null,
+                    "suffix": "",
+                    "thresholds": [
+                        {
+                            "$$hashKey": "object:310",
+                            "color": "#1F60C4",
+                            "state": 0,
+                            "value": 0
+                        }
+                    ],
+                    "unitFormat": "short"
+                },
+                {
+                    "$$hashKey": "object:291",
+                    "clickThrough": "/graph/d/mongodb-instance-summary/mongodb-instance-summary?from=$__from&to=$__to&var-service_name=${__cell_name}",
+                    "colors": [
+                        "#299c46",
+                        "#e5ac0e",
+                        "#bf1b00",
+                        "#ffffff"
+                    ],
+                    "decimals": 0,
+                    "enabled": true,
+                    "label": "OVERRIDE 2",
+                    "metricName": "/.*   /",
+                    "newTabEnabled": true,
+                    "operatorName": "current",
+                    "prefix": "MongoDB - ",
+                    "sanitizeURLEnabled": false,
+                    "scaledDecimals": null,
+                    "suffix": "",
+                    "thresholds": [
+                        {
+                            "$$hashKey": "object:326",
+                            "color": "#8F3BB8",
+                            "state": 0,
+                            "value": 0
+                        }
+                    ],
+                    "unitFormat": "short"
+                },
+                {
+                    "$$hashKey": "object:292",
+                    "clickThrough": "/graph/d/postgresql-instance-summary/postgresql-instance-summary?from=$__from&to=$__to&var-service_name=${__cell_name}",
+                    "colors": [
+                        "#299c46",
+                        "#e5ac0e",
+                        "#bf1b00",
+                        "#ffffff"
+                    ],
+                    "decimals": 0,
+                    "enabled": true,
+                    "label": "OVERRIDE 3",
+                    "metricName": "/.*  /",
+                    "newTabEnabled": true,
+                    "operatorName": "current",
+                    "prefix": "PostgreSQL - ",
+                    "sanitizeURLEnabled": false,
+                    "scaledDecimals": null,
+                    "suffix": "",
+                    "thresholds": [
+                        {
+                            "$$hashKey": "object:372",
+                            "color": "#CA95E5",
+                            "state": 0,
+                            "value": 0
+                        }
+                    ],
+                    "unitFormat": "short"
+                },
+                {
+                    "$$hashKey": "object:539",
```

Comment:
Bunch of hashKeys, let's get rid of them ))

Reply:
done